### DISCUSSION TITLE
android splash screen

### DISCUSSION THREAD

**Discussion by haki-m** [godotengine/godot-proposals#3994] [Platforms]
All I want to mention here is to add some improvements in the Splashscreen Corner, such as movement and seeing the game load percentage

**Comment by theraot** [comment]
This is how I address this: Make a very light scene that looks just like the splashscreen, and then use background loading to load the actual main scene and transition to it. By virtue of being a very light scene it will load very fast. By virtue of looking just like splashscreen the player does not notice it changed. And then you can add animations, a progress bar for the background loading, and some flare as long as it is not too heavy.
